lib/ship.rb

```diff
@@ -0,0 +1,32 @@
+require 'pry';
+
+
+class Ship
+
+  attr_reader :ship_name, :length
+  attr_accessor :health
+
+  def initialize(ship_name, length)
+    @ship_name = ship_name
+    @length = length
+    @health = @length
+  end
+
+  def health
+    @health
```

Comment:
Can we just set this to `health = @length` and then the subsequent methods will still work calling `health` since it is a method? I'm not sure but just wondering if we have to make `health` and instance variable.

Reply:
I think that would work too. I am also not sure if it has to be an instance variable but when I talked to Megan about my branching issue she saw that health was an instance variable and said it looked good lol.